FlowCrypt/src/main/java/com/flowcrypt/email/security/pgp/PgpKey.kt

```diff
@@ -21,8 +23,37 @@ import org.bouncycastle.openpgp.PGPSignature
 import org.bouncycastle.openpgp.jcajce.JcaPGPPublicKeyRingCollection
 import org.bouncycastle.openpgp.jcajce.JcaPGPSecretKeyRingCollection
 import org.bouncycastle.openpgp.operator.jcajce.JcaKeyFingerprintCalculator
+import org.pgpainless.PGPainless
+import org.pgpainless.util.Passphrase
 
 object PgpKey {
+  fun encryptKey(armored: String, passphrase: String): String {
+    return try {
+      val keys = parse(armored)
+      PGPainless.modifyKeyRing(keys[0].keyRing!! as PGPSecretKeyRing)
+          .changePassphraseFromOldPassphrase(null)
+          .withSecureDefaultSettings()
+          .toNewPassphrase(Passphrase.fromPassword(passphrase))
+          .done()
+          .armorWithFlowcryptHeaders()
+    } catch (ex: Exception) {
+      ""
```

Comment:
@IvanPizhenko Please tell me is there any reason to don't print the `stacktrace` of an exception?

Reply:
@DenBond7 This was done to mimic what the Typescript does - it returns empty string on failure. Of course, we can output the exception. But I am just not sure what is the right way to do it on the Android? Just `println()`? Some logger? Please advice.

Reply:
Looked around - looks like just `e.printStackTrace()` would work good

Reply:
Yes, `e.printStackTrace()` - it will be enough. Anyway, don't need to drop using `e.printStackTrace()` over the whole Java(Kotlin) code if you don't have enough reason to do that. Usually, it helps to debug errors.